server/index.js

```diff
@@ -23,15 +23,20 @@ database.once('connected', () => {
 
 // Model imports
 require('./models/testModel');
+require('./models/userModel');
```

Comment:
Changing how we structure the server. This should be moved to the controller file and use `const User = require('../models/userModel')` instead